docker/Dockerfile

```diff
@@ -1,4 +1,11 @@
-FROM ubuntu:15.04
+# This base image is equivalent to Ubuntu 16.04 but has better defaults
+# and fixes numerous problems with the stock Ubuntu images.
+# read more about it here: https://github.com/phusion/baseimage-docker
+FROM phusion/baseimage:0.9.19
```

Comment:
Strange, I just started from a fresh install of Docker, confirmed no images were present via `docker images -a`, and was able to build the `Dockerfile` in viral-ngs-deploy:master with `ubuntu:15.04`. (`tar -czh . | docker build --rm --build-arg VIRAL_NGS_VERSION=1.15.4 -`) I don't see 15.04 listed on the Docker Hub website either, but the tag is visible via the registry API (https://registry.hub.docker.com/v1/repositories/ubuntu/tags), and I can run `docker pull ubuntu:15.04` (can you?).

Thanks for changing the baseimage back to stock Ubuntu. If we run into issues with it we can certainly try `phusion/baseimage` in the future.